benchmarks/sa.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python3                                                         
+
+import time, argparse
+import numpy as np
+import arkouda as ak
+import random
+import string
+
+TYPES = ('int64', 'float64', 'bool', 'str')
+
+def time_ak_sa( vsize,strlen, trials, dtype):
+    print(">>> arkouda suffix array")
+    cfg = ak.get_config()
+    Nv = vsize * cfg["numLocales"]
+    print("numLocales = {},  num of strings  = {:,}".format(cfg["numLocales"], Nv))
+
+    if dtype == 'str':
+         v = ak.random_strings_uniform(1, strlen, Nv)
+    else:
+        print("Wrong data type")
+    c=ak.suffix_array(v)
+#    print("size of suffix array={}".format(c.bytes.size))    
+#    print("offset/number of suffix array={}".format(c.offsets.size))    
+#    print("itemsize of suffix array={}".format(c.offsets.itemsize))    
+    print("All the random strings are as follows")
+    for k in range(vsize):
+       print("the {} th random tring ={}".format(k,v[k]))    
+       print("the {} th suffix array ={}".format(k,c[k]))    
+       print("")
+    timings = []
```

Comment:
There are a couple places in this PR where print statements are being used. I believe these should be converted into logging statements or raised errors (in the case of `Wrong data type`)

Reply:
The `benchmarks` directory and code might be one of those special cases where prints are ok.  I'm not sure how the benchmarking is run but I think there is some type of aggregation of std-out taking place.  We should probably make a larger/longer-term issue to convert the benchmark output to use a logging framework or module, but for now I think they can get a pass.

Any print statements in the _library_ code should be a no-go.